src/ai/anthropic.py

```diff
@@ -0,0 +1,22 @@
+from typing import Optional
```

Comment:
Not reviewing this as I am assuming no change, just moving code around? (everything in dir)

Reply:
yep!